rpm_requirements.txt

```diff
@@ -0,0 +1,10 @@
+bzip2
+git
+GitPython
+lzop
+mock
+nosync
+python-lxml
+PyYAML
+subversion
```

Comment:
Has `svn` package changed its name to `subversion`?  Which one is correct?  Or are both correct?